js/core/arch/x64/ps2/index.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
```

Comment:
`ps2` is not arch specific, so I would put it in a platform-agnostic folder, for example `js/drivers/ps2`.